src/WPF/WPF.Viewer/Samples/Data/ManageFeatures/ManageFeatures.xaml.cs

```diff
@@ -0,0 +1,443 @@
+
+                // Bind the list of method names to the ComboBox.
+                MethodPicker.ItemsSource = _methodList;
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message, ex.GetType().Name, MessageBoxButton.OK, MessageBoxImage.Error);
+            }
+        }
+
+        private void DamageTable_Loaded(object sender, EventArgs e)
+        {
+            // This code needs to work with the UI, so it needs to run on the UI thread.
+            Dispatcher.Invoke(() =>
+            {
+                // Get the relevant field from the table.
+                ServiceFeatureTable table = (ServiceFeatureTable)sender;
+                Field typeDamageField = table.Fields.First(field => field.Name == AttributeFieldName);
+
+                // Get the domain for the field.
+                CodedValueDomain attributeDomain = (CodedValueDomain)typeDamageField.Domain;
+
+                // Update the ComboBox with the attribute values.
+                DamageTypeDropDown.ItemsSource = attributeDomain.CodedValues.Select(codedValue => codedValue.Name);
+            });
+        }
+
+        private async void OnComboBoxSelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            // Clear all potentially hooked GeoViewTapped events.
+            MyMapView.GeoViewTapped -= MapView_Tapped_CreateFeature;
+            MyMapView.GeoViewTapped -= MapView_Tapped_DeleteFeature;
+            MyMapView.GeoViewTapped -= MapView_Tapped_UpdateAttribute;
+            MyMapView.GeoViewTapped -= MapView_Tapped_UpdateGeometry;
+
+            // Hold a variable for the ComboBox's selected value.
+            ComboBox comboBox = (ComboBox)sender;
+
+            if (comboBox.SelectedIndex == 0)
+            {
+                // Update the label.
+                InstructionLabel.Content = "Tap to add a feature.";
+
+                // Update the currently hooked GeoViewTapped event.
+                MyMapView.GeoViewTapped += MapView_Tapped_CreateFeature;
+
+            }
+            else if (comboBox.SelectedIndex == 1)
+            {
+                // Update the label.
+                InstructionLabel.Content = "Tap to delete a feature.";
+
+                // Update the currently hooked GeoViewTapped event.
+                MyMapView.GeoViewTapped += MapView_Tapped_DeleteFeature;
+            }
+            else if (comboBox.SelectedIndex == 2)
+            {
+                // Update the label.
+                InstructionLabel.Content = "Tap to select a feature, then change its damage type.";
+
+                // Update the currently hooked GeoViewTapped event.
+                MyMapView.GeoViewTapped += MapView_Tapped_UpdateAttribute;
+            }
+            else if (comboBox.SelectedIndex == 3)
+            {
+                // Update the label.
+                InstructionLabel.Content = "Tap to select a feature. Tap again to move it.";
```

Comment:
`Tap an existing feature to select it, tap the GeoView to move it to a new position.`